app/views/steps/stage_1.html.erb

```diff
@@ -0,0 +1,7 @@
+<h2>Step One</h2>
+
+<div class="center">
+  <%= form_for current_user, url: step_path do |f| %>
```

Comment:
These are all HTML forms with no data that submit steps in the flow via a `PATCH` request to `/steps/:step` where `:step` is the number, i.e. `1`, `2`, etc.